src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentState = { ...state };
+  const history = [];
+
+  for (const i of actions) {
```

Comment:
`i`, what do u mean? Index? instance? idk..)
change this name)
```suggestion
  for (const action of actions) {
```